src/Tribe/Event_Cleaner_Scheduler.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+// @todo This handles the cleaning AND scheduling...
```

Comment:
Is this a todo or a consideration about the unclear naming of this class? If the second is true, then either rename the class and remove the todo or just remove the todo.